src/EnergyPlus/HeatBalanceSurfaceManager.cc

```diff
@@ -548,7 +548,19 @@ void InitSurfaceHeatBalance(EnergyPlusData &state)
 
     if (state.dataGlobal->ZoneSizingCalc) GatherComponentLoadsSurfAbsFact(state);
 
-    if (state.dataHeatBalSurfMgr->InitSurfaceHeatBalancefirstTime) DisplayString(state, "Completed Initializing Surface Heat Balance");
+    if (state.dataHeatBalSurfMgr->InitSurfaceHeatBalancefirstTime) {
+        DisplayString(state, "Completed Initializing Surface Heat Balance");
+        for (int zoneNum = 1; zoneNum <= state.dataGlobal->NumOfZones; ++zoneNum) {
```

Comment:
This can be wrapped in `if (state.dataHeatBal->AnyEMPD || state.dataHeatBal->AnyHAMT)` No need to search the surfaces if there isn't any EMPD or HAMT in the entire simulation.